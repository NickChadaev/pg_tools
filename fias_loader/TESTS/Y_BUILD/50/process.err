ЗАМЕЧАНИЕ:  индекс "ie4_as_addr_obj" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_object_level" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_addr_obj" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_adm_hierarchy" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_mun_hierarchy" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_houses" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_addr_obj_params" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_apartments_params" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_carplaces_params" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_houses_params" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_rooms_params" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_steads_params" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "iex_as_houses" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "iex_as_steads" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_steads" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_reestr_objects" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_operation_type" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_as_addr_obj_type" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie2_as_addr_obj" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie1_gap_adr_area" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie2_gap_adr_area" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie3_as_addr_obj" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie4_as_addr_obj" не существует, пропускается
ЗАМЕЧАНИЕ:  индекс "ie2_as_adm_hierarchy" не существует, пропускается
ЗАМЕЧАНИЕ:  отношение "ie3_as_adm_hierarchy" уже существует, пропускается
ЗАМЕЧАНИЕ:  отношение "__adr_area_fias" уже существует, пропускается
ЗАМЕЧАНИЕ:  отношение "__adr_area_fias" уже существует, пропускается
psql:stage_4.5.sql:17: ОШИБКА:  таблица "s" отсутствует в предложении FROM
СТРОКА 2: ...NER JOIN gar_tmp.adr_house_aux x ON (x.id_house = s.id_house...
                                                               ^
psql:stage_4.5.sql:19: ОШИБКА:  текущая транзакция прервана, команды до конца блока транзакции игнорируются
psql:stage_4.5.sql:20: ОШИБКА:  текущая транзакция прервана, команды до конца блока транзакции игнорируются
ЗАМЕЧАНИЕ:  индекс "ie4_as_addr_obj" не существует, пропускается
ЗАМЕЧАНИЕ:  отношение "__adr_area_fias" уже существует, пропускается
ЗАМЕЧАНИЕ:  отношение "__adr_area_fias" уже существует, пропускается
