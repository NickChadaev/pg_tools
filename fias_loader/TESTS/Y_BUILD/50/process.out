 save_gar_version 
------------------
               72
(1 строка)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                72
(1 строка)

CALL
 id_area | id_country |  nm_area   |  nm_area_full   | id_area_type | id_area_parent | kd_timezone | pr_detailed | kd_oktmo |             nm_fias_guid             | dt_data_del | id_data_etalon |  kd_okato   | nm_zipcode |  kd_kladr   | vl_addr_latitude | vl_addr_longitude 
---------+------------+------------+-----------------+--------------+----------------+-------------+-------------+----------+--------------------------------------+-------------+----------------+-------------+------------+-------------+------------------+-------------------
      50 |        185 | Московская | Московская обл. |           32 |                |           2 |           0 | 46000000 | 29251dcf-00a1-4e34-98d4-5c47484a36d4 |             |                | 46000000000 | 143700     | 50000000000 |                  |                  
(1 строка)

CALL
CALL
CALL
 f_addr_area_set_data 
----------------------
                  171
(1 строка)

            fias_guid_new             |            fias_guid_old             |      nm_addr_obj       | addr_obj_type_id | chld_qty_tot | obj_level 
--------------------------------------+--------------------------------------+------------------------+------------------+--------------+-----------
 7983872e-a340-4509-a57c-1cab075ef4da | 74943d27-8ed5-4d49-b16d-a0c9cc83c7f0 | 20275                  |              262 |           20 |         7
 e1989edc-0ab0-41cf-a18a-717e3f5e5073 | 116e6c39-2df4-4eb6-bed5-61713e61f517 | 3-ГО ИНТЕРНАЦИОНАЛА    |              204 |           67 |         8
 c85f5305-9af8-4773-9585-0532fad10b95 | fa04375e-afee-4ab4-8f02-f447eb30fd53 | БЕРЕЗКА                |              318 |            1 |         7
 c85f5305-9af8-4773-9585-0532fad10b95 | be237d9b-5742-40c9-8645-96ebde3f6185 | БЕРЕЗКА                |              318 |            2 |         7
 459132aa-097d-4b35-bd4b-5ac0925d7bd3 | 2ea5a970-d1a9-4e7d-962d-a46b3f8002d9 | ВОЛКОВО                |               55 |           39 |         6
 a0a644ae-2e88-42ab-a3c7-8222d87018cf | fde62bc7-fa8a-4db4-bc1a-909761ea5701 | ВОСТОК                 |              318 |           98 |         7
 4f1406b3-ad60-4b14-b4fc-5a1ded841428 | 440d8ab0-176e-41e4-9233-f642b324d911 | ВОСХОД                 |              318 |           15 |         7
 4f1406b3-ad60-4b14-b4fc-5a1ded841428 | e92a63e6-3ce4-42f1-a57b-6e73f06411a2 | ВОСХОД                 |              318 |          191 |         7
 f04e9191-c034-477e-8cb0-09fab63c7ba8 | b820c37e-7d43-4690-ac76-0348d541efd8 | ВОСХОД-1               |              318 |           63 |         7
 a3f9275e-104e-41c6-af7e-b7446f422e8c | 30cd6a92-e9a6-451d-9669-8e999c7ad10d | ВЫСОЧКИ                |               55 |           78 |         6
 71772618-d486-4d3d-8107-1030bbe5864f | cd4cc19c-8374-4e49-b22b-03f886b9ed63 | ГИДРОТЕХНИК            |              318 |           16 |         7
 c17be0e3-91f4-4ffa-9568-efa34c71583a | 4b4ba573-2f18-487a-b21d-7d02c3855465 | ГОРЬКОГО               |              204 |           11 |         8
 37a70c02-665a-40ea-a4dd-44930abde632 | 4cb53700-5b0c-415f-9e30-9a1d40f214fc | ГУСЁНКИ                |               55 |           21 |         6
 a1b00d59-acce-4310-9abe-598aaa9dac16 | ff59f294-5c45-4ec9-832f-bf8148a36f15 | ДНП БЕЛАГО             |              305 |            0 |         7
 02125131-fa12-4cb6-9e9a-72d15931278b | a792016a-1531-4182-92ff-a3d9bd9800eb | ДНП ЛЕСНОЙ ГОРОДОК-2   |              305 |           17 |         7
 62c6f6ee-fb2c-4716-89c2-de454d0af874 | a90f1161-bae3-40a1-886a-926cb335e077 | ДНП МОРОЗОВО           |              305 |            1 |         7
 7fc74155-683c-4da7-a984-f6857459867e | bf2c035f-560b-418b-925c-181faaabc744 | ДНП РАССВЕТ            |              305 |            5 |         7
 12803241-0e18-45af-80f4-680d6d525fde | 6c4228f0-5690-47cb-8ba2-b79594a9913c | ДНП СОЛНЕЧНЫЙ          |              305 |           46 |         7
 d94410cc-e721-4451-8501-63d7cf388a52 | aa756105-9043-4c24-a328-6a35f95b3e43 | ДОМОСТРОИТЕЛЬ          |              304 |           32 |         7
 4ed31d38-01da-4ba0-bdb5-8ffdb8f3e392 | 6890194f-5db3-467f-814f-dfe2c8cb6763 | ДРУЖБА                 |              318 |           97 |         7
 8aac3910-69d0-4360-8e92-599da05769ff | 8d30fb51-ad68-4885-8fa3-2453ccda7b5d | ДРУЖБА                 |              318 |          354 |         7
 7b0a31a0-8a86-47d8-846b-e3c5ea4d347f | 50e26d3f-4739-4bf2-9fac-303658a6ade6 | ДРУЖБА                 |              318 |           93 |         7
 dbedb8a1-e7b2-42d6-b870-eb70f960215c | ce25bb9a-dd4e-453f-8959-d1ef6675037d | ДРУЖБА                 |              318 |           38 |         7
 30a31a02-fc69-4b6d-b57f-c3036e98bf9a | 57895f3e-abea-4ce2-95e1-7ab845f69a98 | ЗАГОРЬЕ                |              318 |           22 |         7
 92612639-6ba4-48a1-8f2d-022840b76a67 | 1a9028b6-0742-479a-bd43-6f83c13c03cd | ЗАРЯ                   |              318 |          151 |         7
 ef3b8236-a64f-4f5a-b04b-c4d1f11fd7e2 | c4120dcf-2699-405b-ba0c-b32654ff5fa7 | ЗЕМЛЕДЕЛЕЦ             |              250 |            0 |         7
 535ec3c5-0181-4eed-a1a1-226d35604dbc | 2a457c31-9adb-4855-82e1-523a4f379d4b | ЗЕНИТ                  |              318 |           16 |         7
 53f27eb5-a76d-4078-bb88-f493115d8525 | 02f9d648-9289-4715-aa2e-c8723f50f2f9 | ИСТОК                  |              305 |            3 |         7
 52767df9-cc1a-4638-bc13-ae676f5866bc | 9a9ec2a2-30fa-442d-9e3d-daaaa7548103 | КИРОВА                 |              204 |           11 |         8
 08a13689-f20d-4367-938d-88af7dd5341a | 1bc605aa-9601-483d-86f7-c42fd037c4a8 | КЛУБНИЧКА              |              318 |           57 |         7
 f985a89f-be4d-41a1-9e67-866199e3770c | 7598a468-79be-4677-8ade-d5d3f1d65786 | КОЖИНО                 |              318 |            7 |         7
 6bc55608-e64d-4c52-9202-45a4d51b79e3 | ab07f11c-cf0d-41d7-9d86-804cc9f054fe | КОЛОСОК                |              318 |           90 |         7
 45a3cb2d-cc3e-4d52-9581-c0da5ec30720 | e132484a-0d09-45b2-a48e-83ef855976c6 | КОМСОМОЛЬСКАЯ          |              204 |           23 |         8
 d869ae34-f522-48f4-b03b-83e75f80552e | 6ab75139-1af1-45c7-bd09-ba436bf8f2b7 | КОСИНСКОЕ              |              210 |           11 |         8
 d294500d-945a-497f-9bc3-10165729dcbb | 9333d3d6-997e-44c1-a120-b79c74a0a1c4 | КРИСТАЛЛ               |              318 |           67 |         7
 26ac6de1-ba61-42f4-8fd4-8ac4eb8a83b5 | c7035814-33ff-4f81-95a5-a4ef94c21773 | КУЗНЕЦОВО              |               55 |           74 |         6
 637eec5c-1faf-4afb-87fc-78182164fa35 | 110fc779-9fc7-4754-a06a-87e153b5f65f | ЛЕСНАЯ ПОЛЯНА          |              318 |           82 |         7
 46c3ebc3-f05a-4717-938e-ad9ad6d15687 | 4b1ed92d-8a13-4031-9e0e-4b2a90c36c70 | ЛЕСНЫЕ ПОЛЯНЫ          |              318 |           57 |         7
 14abaee9-8a5b-4130-9e86-645161b0cba5 | ea764483-6f68-4f69-9210-9c43a7698da0 | ЛЕСНЫЕ ПРУДЫ           |              305 |            7 |         7
 3bdb29ff-d9c7-4a88-9d8e-835b67d481e1 | a52c7b54-c089-4dea-b612-1600b0df6129 | ЛУГОВОЕ                |              305 |            9 |         7
 8e203e14-48b0-4282-88af-ba4129906146 | 968033e7-87e5-490b-8e86-acfb146f5efb | МЕЛИОРАТОР             |              318 |           59 |         7
 d4723d08-3b3a-4874-bd79-9647114a52e3 | 243b3615-f96f-4f46-bde9-706a0128ae60 | НИВА                   |              250 |            8 |         7
 5225cd2c-f04b-4416-9170-f7db5c2efd4d | b47649cb-666e-4b3a-938d-fe1ec78e75ab | НИКОЛИНЫ ОЗЕРА         |              305 |            6 |         7
 2e6ea694-e33c-4091-bd31-2a812e33a096 | 73d1ec46-0395-45cf-8424-4b7a6f4ab57a | НОСОВИХИНСКОЕ ШОССЕ    |              305 |            0 |         7
 498177be-2c70-4763-aedf-406450017c21 | 2208735a-d693-4999-a9c9-fb4337ba234a | ОКОЛИЦА                |              318 |           76 |         7
 42906197-4d31-4728-9588-91fbbe928ec7 | cdde9810-a34d-4ba1-bbbb-a093e8f0e0cf | ОТДЫХ                  |              318 |            9 |         7
 b69ff0fc-efef-438e-9c0b-82fa62a8eadb | be159978-fae5-478b-85e2-8eda03296845 | ПИОНЕРСКАЯ             |              204 |            6 |         8
 d9838500-ba57-4a01-be29-9a97497d8566 | 8294b136-fe7d-46df-8f44-c7a60b052dea | ПОЛЯНА                 |              318 |           49 |         7
 ba8a79ab-4914-4618-839d-423f14675a6e | 9a936c2f-59dc-4d0c-9340-bae3031271eb | ПОЛЯНКА                |              318 |            0 |         7
 f4000e71-6d1c-437c-b72b-95cb0ec95e7e | 3c55237e-a34b-4da5-a428-7e809a18e618 | ПРОЛЕТАРСКАЯ           |              204 |           11 |         8
 075f11bb-81c0-4fff-9824-e155a8b8eb2f | 67b74474-ccce-4d1f-b62a-9019e11fd911 | ПРОМПЛОЩАДКА N 1       |              304 |           16 |         7
 dcc5bbb1-a303-4563-ac82-5d0a302d1b15 | 4eddfb1b-a568-426f-a084-1a80bb9db883 | РОДНИК                 |              318 |            4 |         7
 55849d20-46b7-4f80-9b83-57d07875a8c7 | bfaf3778-4387-439e-8021-56856ce0a625 | РОДНИК                 |              318 |          162 |         7
 c683ae4e-dcd1-4f61-9aa8-4ebe985b26a9 | 0cf8b042-c6c2-4ceb-8fe1-7dd249a33586 | РОМАШКА                |              318 |           74 |         7
 e8dadad2-6b34-4770-8a5e-f38ef66d8688 | 8e9df847-d554-456c-977c-98687678cff0 | РОТА-АГРО БЛАГОВЕЩЕНЬЕ |              305 |            2 |         7
 2b2ecde1-fc06-4c64-9111-9ece166d26b0 | df6f7321-f0cc-41bf-976f-1bac1d069353 | СЕРГИЕВ ПОСАД-6        |              305 |           12 |         7
 032bef95-f558-4c76-871c-2bbec6b86991 | e6dcde9a-06a8-4051-b7bf-a082078e7069 | СНТ ВОСХОД             |              305 |            7 |         7
 0c1bd37b-27eb-4623-a2c9-f4f2d1877785 | f9b9a6be-23b6-4843-97f8-ae94a2b4cd50 | СНТ ЗАРЯ               |              305 |           20 |         7
 82e27793-3718-4eb2-836c-dddb18674731 | 94157858-86b8-452d-83da-df2f39f26b56 | СНТ РАССВЕТ            |              305 |           26 |         7
 31766343-fbd4-413e-9958-af4e516c9ccd | 079e764b-740e-4fe2-b314-78e029e88c9c | СНТ СВЯЗИСТ            |              305 |            1 |         7
 e7972ffe-645a-4c70-952b-feee9356f927 | 46bef183-e6c7-417b-bff6-49a171e6841f | СНТ СТРОИТЕЛЬ          |              305 |           14 |         7
 ba4ad9af-e032-4172-91be-e6face91bd14 | 6a31c502-b0c1-4d23-a4f0-5b4d37f5e8b7 | СОДРУЖЕСТВО            |              318 |          164 |         7
 30882d02-2e2b-40e2-8e44-b3f30f866793 | 1b112e6e-5cef-407f-9b2d-e1ee95ebc026 | ТЕКСТИЛЬЩИК            |              318 |          200 |         7
 2f3c1a48-e78a-4a86-90bd-07765f0081aa | 92672c90-4c21-46e7-a08f-9292825bac6c | ТЕШИЛОВО               |               55 |           31 |         6
 dd99df6f-c30e-4c4e-9a20-d8e381807fb8 | b3b3e1dc-2e6d-46fe-b51e-2abca18e72ce | ТИХАЯ РЕЧКА            |              305 |            0 |         7
 884e0862-aa3b-463d-9ac3-06bc2b1fdab8 | e4404769-507d-488e-9211-390769a182a0 | ТРУД                   |              318 |           10 |         7
 884e0862-aa3b-463d-9ac3-06bc2b1fdab8 | 5470bf73-8900-4e4b-935c-d2724ea3ca3e | ТРУД                   |              318 |           57 |         7
 b0aa0895-e596-4a25-a0aa-0c69c83f0f9e | 80a6adb4-a120-4f45-9a50-646ee565d37a | УВАРОВО                |               55 |           62 |         6
 4314bca0-4bfd-41bb-9146-1a0ff1af7a12 | 3ece5c23-5913-4e59-9dd2-dc6b05fe1c84 | УГОЛЕК                 |              318 |           22 |         7
 ecf6510a-e8bf-480b-a637-d027670a3385 | 217176cd-4e0d-4ebc-b694-f8f112a525d6 | ШИШКИНО                |               55 |          235 |         6
 f7d7063b-4f9c-4fa9-af73-f7d6e622cd0b | f5c99eb7-93d2-4756-a2de-c24ec00452e3 | ЭНЕРГЕТИК              |              318 |          139 |         7
 c3b17288-b566-43c2-89e0-95e5a95fb68b | f588e7b5-c2dd-4462-bdd4-ee0bc18628f1 | ЮЖНОЕ                  |              318 |           51 |         7
 2316face-0946-43f7-a979-4740bdb963cd | 6b214642-073b-49c5-9812-6f3ee3e0bef8 | ЮЖНЫЙ                  |              261 |            6 |         7
 63837530-3bfa-4d70-8620-202d2fe58250 | 698268eb-c808-4135-af6a-e97a8dcd5881 | ЯСНАЯ ПОЛЯНА           |              318 |           38 |         7
(74 строки)

 id_addr_obj | id_addr_parent |              fias_guid               |           parent_fias_guid           |       nm_addr_obj        | addr_obj_type_id | addr_obj_type | obj_level |           level_name            | region_code | area_code | city_code | place_code | plan_code | street_code | change_id | prev_id | oper_type_id | oper_type_name | start_date |  end_date  | id_lead |                tree_d                | level_d 
-------------+----------------+--------------------------------------+--------------------------------------+--------------------------+------------------+---------------+-----------+---------------------------------+-------------+-----------+-----------+------------+-----------+-------------+-----------+---------+--------------+----------------+------------+------------+---------+--------------------------------------+---------
      846056 |         825559 | 04c448da-7141-49cb-960f-9ea70f715929 | b0aa0895-e596-4a25-a0aa-0c69c83f0f9e | ДНП Ясенево              |              305 | тер.          |         7 | Элемент планировочной структуры | 50          | 0         | 1         | 175        | 6         | 0           | 206730520 | 1025811 |           20 | Изменение      | 2021-08-11 | 2079-06-06 |  846056 | {807356,811082,825559,846056}        |       4
      844206 |         825559 | ce5a2567-c02a-49f1-8a2e-2dccc9f95712 | b0aa0895-e596-4a25-a0aa-0c69c83f0f9e | ДНП Ясенево              |              305 | тер.          |         7 | Элемент планировочной структуры | 50          | 0         | 1         | 251        | 1         | 0           | 507478127 | 1022126 |           20 | Изменение      | 2023-07-28 | 2079-06-06 |  846056 | {807356,811082,825559,844206}        |       4
      871190 |         862067 | 745e292a-739d-44dd-869b-b1b4d09538e4 | 1edc94f7-98d0-4579-bf08-976cd6f9bc39 | Дачная                   |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 3         | 0          | 167       | 172         |   2410087 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  871190 | {807356,862378,866498,862067,871190} |       5
      875792 |         862067 | f2c591bf-1989-4362-999d-81571d0b4a34 | 1edc94f7-98d0-4579-bf08-976cd6f9bc39 | Железнодорожная          |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 3         | 0          | 167       | 173         |   2423465 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  875792 | {807356,862378,866498,862067,875792} |       5
      868589 |         862067 | ea446de6-fb3d-40bb-90b8-64ca46ceba95 | 1edc94f7-98d0-4579-bf08-976cd6f9bc39 | Лесная                   |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 3         | 0          | 167       | 169         |   2401337 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  868589 | {807356,862378,866498,862067,868589} |       5
      864313 |         862067 | 02a7c4b8-ed6d-4bd9-a7b5-c244dbb1b7bc | 1edc94f7-98d0-4579-bf08-976cd6f9bc39 | Садовая                  |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 3         | 0          | 167       | 170         |   2385929 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  864313 | {807356,862378,866498,862067,864313} |       5
      865528 |         862067 | 8ca5dd2d-f0d0-485f-ae6c-70c48b880e92 | 1edc94f7-98d0-4579-bf08-976cd6f9bc39 | Центральная              |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 3         | 0          | 167       | 171         |   2390196 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  865528 | {807356,862378,866498,862067,865528} |       5
      867100 |         862067 | d41bd343-f7a3-4509-acd6-596c4ba47f45 | 1edc94f7-98d0-4579-bf08-976cd6f9bc39 | Школьная                 |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 3         | 0          | 167       | 168         |   2395881 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  867100 | {807356,862378,866498,862067,867100} |       5
      867163 |         862378 | de1588bd-1754-4b55-8b1a-d9ac1183653f | 6ae8ad9e-aa6c-4849-92d8-8e637059c9b0 | ДНТ Баковка-Набережная   |              110 | тер           |         6 | Населенный пункт                | 50          | 42        | 0         | 0          | 1743      | 0           |   2396175 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  867163 | {807356,862378,867163}               |       3
      864577 |         862378 | 86997df8-36f7-4a70-9eb6-b858c7902a5e | 6ae8ad9e-aa6c-4849-92d8-8e637059c9b0 | ДСК Сельскохозяйственный |              110 | тер           |         6 | Населенный пункт                | 50          | 42        | 0         | 0          | 1745      | 0           |   2386868 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  864577 | {807356,862378,864577}               |       3
      864485 |         864577 | fd0a7955-a913-4eb1-8b6a-9a0f2b225f7f | 86997df8-36f7-4a70-9eb6-b858c7902a5e | 1-й Пионерский           |              163 | пер           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1758        |   2386532 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  864485 | {807356,862378,864577,864485}        |       4
      863749 |         864577 | 8c3a6254-8ee8-42fe-9b01-7a5aab57c866 | 86997df8-36f7-4a70-9eb6-b858c7902a5e | 2-й Пионерский           |              163 | пер           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1762        |   2383902 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  863749 | {807356,862378,864577,863749}        |       4
      870356 |         864577 | eec9ca98-379d-4a48-bca6-996b0644986c | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Комсомольский            |              163 | пер           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1746        |   2407441 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  870356 | {807356,862378,864577,870356}        |       4
      864545 |         864577 | ab5fb0c1-ea62-4e4e-8139-1941fb3b5984 | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Маяковского              |              163 | пер           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1757        |   2386806 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  864545 | {807356,862378,864577,864545}        |       4
      870073 |         864577 | a75a31bc-b7ee-4c60-9e03-9495730cf484 | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Мичуринский              |              163 | пер           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1751        |   2406496 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  870073 | {807356,862378,864577,870073}        |       4
      867398 |         864577 | dc626929-0815-40e4-b22c-dd2c99edc1ff | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Овражный                 |              163 | пер           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1752        |   2397117 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  867398 | {807356,862378,864577,867398}        |       4
      863214 |         864577 | abd7e910-17ce-4c2e-872e-66fd218af7ec | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Пушкинский               |              163 | пер           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1760        |   2381940 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  863214 | {807356,862378,864577,863214}        |       4
      865030 |         864577 | db01cdbe-9be3-4eab-a890-fc6208f825eb | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Рыбалко                  |              163 | пер           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1749        |   2388349 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  865030 | {807356,862378,864577,865030}        |       4
      866316 |         864577 | c1f81f0f-0dd7-4e1d-824e-41aa26d417c6 | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Лесной                   |              203 | туп           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1753        |   2393022 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  866316 | {807356,862378,864577,866316}        |       4
      866086 |         864577 | 696b10a6-e922-477a-86b3-c146bf46b2ce | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Сетуньский               |              203 | туп           |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1747        |   2392174 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  866086 | {807356,862378,864577,866086}        |       4
      864402 |         864577 | cd666f0b-bc9e-48ab-957e-97fe2724768d | 86997df8-36f7-4a70-9eb6-b858c7902a5e | 1905 года                |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1761        |   2386325 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  864402 | {807356,862378,864577,864402}        |       4
      870206 |         864577 | 714717f4-624a-4d0d-848e-b49ac5e94d3f | 86997df8-36f7-4a70-9eb6-b858c7902a5e | 1-го Мая                 |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1750        |   2407108 | 1075546 |           20 | Изменение      | 2019-05-16 | 2079-06-06 |  870206 | {807356,862378,864577,870206}        |       4
      864891 |         864577 | 2c3aaeea-0bfa-4a11-98f2-11b97fc450de | 86997df8-36f7-4a70-9eb6-b858c7902a5e | им. Иосифа Кобзона       |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1756        |   2387923 | 1066650 |           20 | Изменение      | 2019-06-28 | 2079-06-06 |  864891 | {807356,862378,864577,864891}        |       4
      865306 |         864577 | d60d4dea-e700-4c36-b420-4f1078d45878 | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Нагорная                 |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1748        |   2389371 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  865306 | {807356,862378,864577,865306}        |       4
      866747 |         864577 | 0c5e572b-5e67-4b5f-8e53-89a6fb725c54 | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Полевая                  |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1754        |   2394594 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  866747 | {807356,862378,864577,866747}        |       4
      863306 |         864577 | d531fd22-c871-446f-9fa1-c21ffc0959be | 86997df8-36f7-4a70-9eb6-b858c7902a5e | Буденновское             |              210 | ш             |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1745      | 1759        |   2382304 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  863306 | {807356,862378,864577,863306}        |       4
      862067 |         866498 | 1edc94f7-98d0-4579-bf08-976cd6f9bc39 | 50341359-842b-4ba1-b7f2-398a7b72f829 | Красная Горка            |              262 | кв-л          |         7 | Элемент планировочной структуры | 50          | 42        | 3         | 0          | 167       | 0           | 151056682 | 1061895 |           50 | Переподчинение | 2020-05-30 | 2079-06-06 |  862067 | {807356,862378,866498,862067}        |       4
      870235 |         867163 | 5c59c932-e008-4803-ac90-50157b702e5a | de1588bd-1754-4b55-8b1a-d9ac1183653f | Набережная               |              204 | ул            |         8 | Элемент улично-дорожной сети    | 50          | 42        | 0         | 0          | 1743      | 1744        |   2407029 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |  870235 | {807356,862378,867163,870235}        |       4
(28 строк)

 id_area | id_country |  nm_area   |  nm_area_full   | id_area_type | id_area_parent | kd_timezone | pr_detailed | kd_oktmo |             nm_fias_guid             | dt_data_del | id_data_etalon |  kd_okato   | nm_zipcode |  kd_kladr   | vl_addr_latitude | vl_addr_longitude 
---------+------------+------------+-----------------+--------------+----------------+-------------+-------------+----------+--------------------------------------+-------------+----------------+-------------+------------+-------------+------------------+-------------------
      50 |        185 | Московская | Московская обл. |           32 |                |           2 |           0 | 46000000 | 29251dcf-00a1-4e34-98d4-5c47484a36d4 |             |                | 46000000000 | 143700     | 50000000000 |                  |                  
(1 строка)

 f_xxx_obj_seq_crt 
-------------------
        5000438698
       15000438698
(2 строки)

 f_set_params_value 
--------------------
            5598523
(1 строка)

 f_xxx_adr_area_set_data 
-------------------------
                   72028
(1 строка)

 f_xxx_adr_house_set_data 
--------------------------
                  1811940
(1 строка)

 f_xxx_obj_fias_set_data 
-------------------------
                   28106
                   43922
                 1811934
(3 строки)

